exercises/practice/poker/tests/poker.rs

```diff
@@ -26,6 +26,16 @@ fn test_single_hand_always_wins() {
     test(&["4S 5S 7H 8D JC"], &["4S 5S 7H 8D JC"])
 }
 
+#[test]
+#[ignore]
+fn test_duplicate_hands_always_tie() {
+    let input = &["3S 4S 5D 6H JH", "3S 4S 5D 6H JH", "3S 4S 5D 6H JH"];
+    assert_eq!(
+        &winning_hands(input).expect("This test should produce Some value",),
```

Comment:
```suggestion
        &winning_hands(input).expect("This test should produce Some value"),
```

The extra comma is legal there but looks weird to me.

Reply:
Extracted this to https://github.com/exercism/rust/issues/1144.